### ISSUE TITLE
broken in vertical

### ISSUE THREAD

**Issue by clepdn** [antsstyle/nftcryptoblocker#4]
iOS 15.3.1 iphone 12 Pro in safari

The collapsible buttons cover the page when rotated vertically but is fine in landscape

![9D5C4DA1-21BD-468F-8EB3-04E97D639EE6](https://user-images.githubusercontent.com/25444923/157046460-e6527c5f-0880-4626-bf1e-48660576f4f9.png)

**Comment by antsstyle** [comment]
Yeah, that's an issue I've been meaning to fix for a couple of days. Will get around to it soon ^^

**Comment by antsstyle** [comment]
I've identified the error that causes this, shouldn't be too hard to fix. I should have it done fairly soon; I'm doing some other code improvements alongside it, so it should be all fixed up in a week or two.
